cmd/bb_worker/main.go

```diff
@@ -75,6 +76,12 @@ func main() {
 		filePool = re_filesystem.NewDirectoryBackedFilePool(filePoolDirectory)
 	}
 
+	sharedActionQueueSize := configuration.SharedActionQueueSize
+	if sharedActionQueueSize == 0 {
+		sharedActionQueueSize = 255
```

Comment:
Please don't add defaults like these. Current Buildbarn style is to refrain from setting default size constants. This is done intentionally. The size of this queue would strongly depend on how people deploy Buildbarn. For very small workers, 255 would be far too high.

Reply:
In that case would you like me to default it to one? We can't default it to zero as it will cause the queue to never run. Alternatively I can make it whatever this number is +1 to get around this.

Reply:
Sure, or call `log.Fatal()` in case it's `<= 0`, so that the user comes up with an explicit value for it.

Reply:
Done.